docs/glossary.rst

```diff
@@ -0,0 +1,17 @@
+Glossary
+========
+
+.. glossary::
+
+    deck/input deck
+        Inputs provided to reservoir simulators such as Eclipse or OPM-flow.
+        Usually a .DATA file pointing to other include files. One deck points
+        to multiple include files.
+
+    include files
+        Files that provide inputs to reservoir simulators by using the INCLUDE statement
+        in input decks. By convention, these files often have the extension .INC (generally)
+        or .GRDECL (for files included into the grid section).
```

Comment:
I think it is actually more common to see these as lower-case, but that could easily be an Equinor/FMU-thing.